controllers/dscinitialization/resources/observability/alertmanager/alertmanagerconfig.tmpl.yaml

```diff
@@ -0,0 +1,54 @@
       <h1>Notifications</h1>                                             </td>                                         </tr>                                     </table>                                 </td>                             </tr>                             <!-- end head -->                              <!-- Body section -->                             <tr>                                 <td class="rh-body">                                     <table class="rh-content" role="presentation" align="center" border="0" cellpadding="0"                                         cellspacing="0" width="100%">                                         {{ range .Alerts.Firing }}                                             {{ if .Annotations.message }}                                                 <tr>                                                     <td class="rh-content__block">                                                         <div class="rh-alert__block">                                                             <h4 class="fh-inline-title"> {{ .Annotations.summary }}</h4>                                                              {{ .Annotations.message }}                                                         </div>                                                     </td>                                                 </tr>                                             {{ end }}                                                      {{ end }}                                         <tr><td class="rh-content__block"><div class="rh-spacer"></div></td></tr>                                     </table>                                 </td>                             </tr>                             <!-- end body section -->                              <!-- Footer -->                             <tr>                                 <td class="rh-footer">                                     <table role="presentation" border="0" align="center" cellpadding="0" cellspacing="0"                                         width="100%">                                         <tr>                                             <td class="rh-footer__text">                                                 This email was sent by Red Hat OpenShift Data Science                                             </td>                                         </tr>                                     </table>                                 </td>                             </tr>                             <!-- end footer -->                              <!-- Ensure padding bottom - don''t remove empty row -->                             <tr>                                 <td>                                 </td>                             </tr>                             <!-- leave - empty row -->                          </table>                     </td>                 </tr>             </table>         </center>     </body>     </html>     {{ end }}'
```

Comment:
https://github.com/opendatahub-io/opendatahub-operator/blob/1c160a4aaf25e7e2afd8c0a22333691d3599c10e/controllers/dscinitialization/resources/observibility/alertmanager/alertmanagerconfig.yaml#L33 check the way how this is done. 
i have not tested it but i think it should work (hopefully)

Reply:
this seems better, but I dont understand how the mapping happens.
Also how do we create a notification email to test it 😅

Reply:
this is interesting. But I have this question:
Can we create an alertmanager instance ourselves? I thought the `MonitoringStack` CR took care of it 🤔

Reply:
correct. MonitoringStack need to change .spec.AlertmanageConfig.disable to true and we can create AlertManager CR by ourselves.

Reply:
in this case, we should make sure that the config map is deployed first before other crs are deployed right?

Reply:
this should not be a too big problem, a race condition could happen but all three should be created almost the same time from one folder. similar to how we deploy old prom stack, done by configmap. 
we will need to test this for sure